.github/workflows/vendor_gardener.yaml

```diff
@@ -24,8 +24,8 @@ jobs:
       run: make revendor
```

Comment:
Not related to this PR but I found it while I was exploring github actions/dependabot. IMO, you can drop https://github.com/gardener/gardener-extension-os-gardenlinux/blob/996c5bc3df3fc7996af9c075603c649bacad6fa8/.github/workflows/vendor_gardener.yaml#L6-L8.
See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request. My understanding is that `branches` is the target branch - the branch where the commits from the PR will be merged. In our case, this does not make sense as we don't expect anyone to open PRs against `dependabot/go_modules/github.com/gardener/**`.